### ISSUE TITLE
Disable resize

### ISSUE THREAD

**Issue by CW12** [Reklino/angular-resizable#64]
I'd like an option to disable the resize functionality. I.e. what I'm looking for is an element that is resizable when a condition is true and not when it's false (so I can use an "Edit mode" button, and my div can only be resized when editmode == true)

**Comment by Halomaple** [comment]
Yes, this will be quite useful when using it to wrap a side panel whose resizing functionality suppose to be configurable.

**Comment by CW12** [comment]
Hi Halo, I hired someone to make this mod for me - please see Kapil's fork: https://github.com/kapil2708/angular-resizable - it might not do exactly what you need but may be a start.
